UI/window-basic-settings-stream.cpp

```diff
@@ -1272,6 +1356,22 @@ static QString get_adv_fallback(const QString &enc)
 	return "obs_x264";
 }
 
+static QString get_adv_audio_fallback(const QString &enc)
+{
+	const char *codec = obs_get_encoder_codec(QT_TO_UTF8(enc));
+
+	if (strcmp(codec, "aac") == 0)
+		return "ffmpeg_opus";
```

Comment:
I assume this is meant to fall back to opus if AAC is used and AAC if opus is used, wouldn't it make sense to try other AAC encoders first before switching to a diferent codec?

Reply:
If the unsupported codec is AAC, why return an AAC encoder ?

We should fallback to the opus encoder.